### PR TITLE
Demo practice

### PR DESCRIPTION
**Pull request by ademcigdem**

### CODE REVIEW

File: src/test/java/HelloWorld.java [hunk 1]
```diff
@@ -1,5 +1,9 @@
 public class HelloWorld {
     public static void main(String[] args) {
-        System.out.println("Hello World");
+
+        System.out.println("Hello new World");
```
Comment (HarunGumusPhD) [inline src/test/java/HelloWorld.java:4]: why you are add this

**Review by HarunGumusPhD — COMMENTED** [review]
ok

File: src/test/java/HelloWorld.java (hunk 1 above)
Comment (ademcigdem) [reply src/test/java/HelloWorld.java:4]: It's all good

File: src/test/java/HelloWorld.java [hunk 2]
```diff
@@ -1,5 +1,9 @@
 public class HelloWorld {
     public static void main(String[] args) {
-        System.out.println("Hello World");
+
```
Comment (idrisdogruoz) [inline src/test/java/HelloWorld.java:3]: why is that

**Review by idrisdogruoz — COMMENTED** [review]
?

**Review by nevzattalhaozcan — APPROVED** [review]

**Review by hasanaliaksoy — APPROVED** [review]
ok

**Review by idrisdogruoz — APPROVED** [review]

**Review by HarunGumusPhD — APPROVED** [review]
all is well

**Pull request merged**